File: app/src/main/java/de/christinecoenen/code/zapp/model/json/JsonChannelModel.java
package de.christinecoenen.code.zapp.model.json;


import com.google.gson.annotations.SerializedName;

/**
 * Transitional model to store parsed json data.
 */
class JsonChannelModel {
	@SuppressWarnings("unused")
	@SerializedName("id")
	String id;

	@SuppressWarnings("unused")
	@SerializedName("name")
	String name;

	@SuppressWarnings("unused")
	@SerializedName("stream_url")
	String streamUrl;

	@SuppressWarnings("unused")
	@SerializedName("logo_name")
	String logoName;

	@SuppressWarnings("unused")
	@SerializedName("subtitle")
	String subtitle;

	@SuppressWarnings("unused")
	@SerializedName("color")
	String color;
}

File: app/src/main/java/de/christinecoenen/code/zapp/model/json/JsonChannelModel.kt
package de.christinecoenen.code.zapp.model.json


import com.google.gson.annotations.SerializedName

/**
 * Transitional model to store parsed json data.
 */
internal data class JsonChannelModel(

	@SerializedName("id")
	val id: String,

	@SerializedName("name")
	val name: String,

	@SerializedName("stream_url")
	val streamUrl: String,

	@SerializedName("logo_name")
	val logoName: String,

	@SerializedName("subtitle")
	val subtitle: String? = null,

	@SerializedName("color")
	val color: String

)
